cmds/identityd/main.go

```diff
@@ -461,7 +461,7 @@ func registerNode(nodeID pkg.Identifier, farmID pkg.FarmID, version string, stor
 
 	_, err := store.RegisterNode(nodeID, farmID, version, loc)
 	if err != nil {
-		log.Error().Err(err).Send()
+		log.Error().Msgf("%v, farmID: %d is not valid", err, farmID)
```

Comment:
you do not know if the error mean the farmer ID is invalid. It could be anything. 
What assumption make you think this is the only possible reason for the error ?

Reply:
True, maybe the API should return a description of the error.